cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -54,6 +57,24 @@ def new_instance(provider):
         pass
 
 
+@pytest.fixture(scope='module')
+def volume(appliance, provider):
+    collection = appliance.collections.volumes
+    storage_manager = '{} Cinder Manager'.format(provider.name)
+    volume = collection.create(name=fauxfactory.gen_alpha(),
+                               storage_manager=storage_manager,
+                               tenant=provider.data['provisioning']['cloud_tenant'],
+                               size=VOLUME_SIZE,
+                               provider=provider)
+    yield volume
+
+    try:
```

Comment:
Required: you don't need try/except here, volume.exists returns bool, and if we cannot delete volume better the teardown failed